gammapy/data/event_list.py

```diff
@@ -552,9 +552,11 @@ def plot_offset2_distribution(self, ax=None, center=None, **kwargs):
             center = self._plot_center
 
         offset2 = center.separation(self.radec) ** 2
+        max2 = np.percentile(offset2, q=98)
```

Comment:
I think it would be good to make the quantile a parameter to `plot_offset2_distribution()`. I don't expect it to be changed much, but just in case it good to avoid the hard-coded number...